packages/protocol/contracts/L1/hooks/AssignmentHook.sol

```diff
@@ -112,7 +112,7 @@ contract AssignmentHook is EssentialContract, IHook {
         // Ether or ERC20 tokens.
         if (assignment.feeToken == address(0)) {
             // Paying Ether
-            _blk.assignedProver.sendEther(proverFee, MAX_GAS_PAYING_PROVER);
+            _blk.assignedProver.sendEtherAndVerify(proverFee, MAX_GAS_PAYING_PROVER);
```

Comment:
should be allowed to fail I think, especially because of the gas limit

Reply:
Emitting an event in case this payment fails.